### ISSUE TITLE
NoMethodError in Spree::Admin::Orders#index

### ISSUE THREAD

**Issue by voodee** [spree-contrib/spree_static_content#192]
> Showing ~/.rvm/gems/ruby-2.2.1/gems/spree_backend-3.0.0/app/views/spree/layouts/admin.html.erb where line # 40 raised:

> undefined method `%' for :many:Symbol

### Extracted source (around line # 18):
```
        lambda do |n|
          n ||= 0
>          mod10 = n % 10
          mod100 = n % 100

          if mod10 == 1 && mod100 != 11
```

### GemFile
```
gem 'spree', '3.0.0'
gem 'spree_gateway', github: 'spree/spree_gateway', branch: '3-0-stable'
gem 'spree_auth_devise', github: 'spree/spree_auth_devise', branch: '3-0-stable'
gem 'spree_i18n', github: 'spree-contrib/spree_i18n', branch: '3-0-stable'
gem 'spree_static_content', github: 'spree-contrib/spree_static_content', branch: '3-0-stable'
```

### application.rb
```
    config.i18n.default_locale = :ru
```

**Comment by sergey-koba-mobidev** [comment]
the same!

**Comment by devsli** [comment]
Upstream issue: spree/spree#5842
Fixed in master branch with 3d870caf1078117a4ee5339aa134a4e104629887
Backport PR #195

**Comment by futhr** [comment]
@voodee @sergey-koba-mobidev @devsli Think this has been resolved with cbeaceb1e799568572855777c8b406e900abf571
